0.2-Elliptic-Curve-Math.ipynb

```diff
@@ -97,28 +97,30 @@
    "outputs": [],
    "source": [
     "# int() operations\n",
-    "# get 2 random numbers a,b\n",
+    "# get 2 random numbers a, b\n",
     "a = random.randrange(1, SECP256K1_ORDER)\n",
     "b = random.randrange(1, SECP256K1_ORDER)\n",
     "print(\"a = {}\".format(a))\n",
     "print(\"b = {}\".format(b))\n",
     "\n",
-    "# use simple addition for a+b but modulo the result to make sure we stay within the SECP256K1_ORDER order\n",
+    "# use simple addition for a + b but modulo the result to make sure we stay within the SECP256K1_ORDER order\n",
     "ab = (a + b) % SECP256K1_ORDER\n",
     "print(\"\\na + b = {}\\n\".format(ab))\n",
     "\n",
     "# ECKey() operations\n",
-    "# use our bytes to instantiate ECKey instances. True means we are using compressed keys\n",
-    "a_key = ECKey().set(a, True)\n",
-    "b_key = ECKey().set(b, True)\n",
+    "# Use the set() method to instantiate ECKey instances\n",
+    "a_key = ECKey().set(a)\n",
+    "b_key = ECKey().set(b)\n",
     "\n",
     "ab_key = a_key + b_key\n",
-    "print(\"a_key.secret = {}\".format(a_key.secret))\n",
-    "print(\"b_key.secret = {}\".format(b_key.secret))\n",
-    "print(\"\\nab_key.secret = {}\\n\".format(ab_key.secret))\n",
     "\n",
-    "# Ensure equivalency.\n",
-    "assert ab_key.as_int() == ab\n",
+    "print(\"a_key private key = {}\".format(a_key))\n",
+    "print(\"b_key private key = {}\".format(b_key))\n",
+    "print(\"\\nab_key private key = {}\\n\".format(ab_key))\n",
+    "\n",
+    "# Check operations are equivalent\n",
+    "# Use .secret to retrieve the ECKey's private key \n",
```

Comment:
why not `as_int()`?

Reply:
I'd like to remove `as_int()` and `from_int()` entirely since they're redundant with `.secret` and `set(<int>)`. What do you think?

Reply:
hmm It's a matter of convention I think.
Some languages are more for accessing variables others are more for using functions to access variables.
But I think that python wise you're correct, "Pythonic" code is more using attributes and less functions (which is why there's the `@property` attribute)